src/scripts/movies.js

```diff
@@ -163,8 +198,12 @@ define(["layoutManager", "userSettings", "events", "libraryBrowser", "alphaPicke
                 StartIndex: 0,
                 Limit: 100,
                 ParentId: params.topParentId
-            };
-        "favorites" === options.mode && (query.IsFavorite = !0), query = userSettings.loadQuerySettings(savedQueryKey, query), self.showFilterMenu = function() {
+            },
+            isLoading;
+        if(options.mode === "favorites")
```

Comment:
```suggestion
        if (options.mode === "favorites") query.IsFavorite = true;
```
Please don't split such one-liners, too error prone on future editing.